index.html

```diff
@@ -2757,17 +2779,31 @@ <h3>Algorithm</h3>
                       either <code>@language</code>, <code>@index</code>, <code>@id</code>, or <code>@type</code>
                       based on the contents of <var>container</var>, as <var>var</var>, and <code>true</code>
                       for <var>vocab</var>.</li>
+                    <li class="changed">Set <var>index key</var> to the value of <a>index mapping</a> in
+                      the <a>term definition</a> associated with <var>item active property</var> in <var>active context</var>,
+                      or <code>@index</code>, if no such value exists.</li>
                     <li>If <var>container</var> includes <code>@language</code> and
                       <var>expanded item</var> contains a
                       <code>@value</code> <a>member</a>, then set <var>compacted item</var>
                       to the value associated with its <code>@value</code> <a>member</a>.
                       Set <var>map key</var> to the value of <code>@language</code> in <var>expanded item</var>, if any.</li>
-                    <li>If <var>container</var> includes <code>@index</code> set <var>map key</var> to the value of <code>@index</code> in <var>expanded item</var>, if any,
+                    <li>Otherwise, if <var>container</var> includes <code>@index</code>
+                      and <var>index key</var> is <code>@index</code>,
+                      set <var>map key</var> to the value of <code>@index</code> in <var>expanded item</var>, if any,
                       and remove <var>container key</var> from <var>compacted item</var>.</li>
-                    <li class="changed">If <var>container</var> includes <code>@id</code>, set
+                    <li class="changed">Otherwise, if <var>container</var> includes <code>@index</code>
+                      and <var>index key</var> is not <code>@index</code>,
+                      Set <var>map key</var> to the first value of <var>container key</var> in <var>compacted item</var>, if any.
+                      If there are remaining values in <var>compacted item</var>
```

Comment:
This sentence is really not clear for me. It seems to me that `compacted container` should be replaced by `container key` and that `compacted value` should be replaced by `compacted item`.
Indeed:
* `compacted item` is never defined, AFAICT
* `compacted value` is used in other parts of the algorithm, but does not seem to have a value in this branch

The same sentence appears later (line 2808), and should be fixed the same way, if I'm correct.

Reply:
__compacted item__ is defined in 9.7.5. (You can select the variable in HTML to find all the places it's used in the algorithm, which should start with its definition.)

It should be using __compacted item__ and __compacted key__ consistently. I'll fix.

Reply:
Sorry, I meant "`compacted container` is never defined"